api/models/User.js

```diff
@@ -18,9 +22,14 @@ var userSchema = new mongoose.Schema({
  *     .then(user => console.log(user))
  *     .catch(error => console.error(error));
  */
-userSchema.statics.create = function(name) {
+
+userSchema.statics.create = function(name, GoogleID) {
```

Comment:
That’s not possible for these static functions because of `new this` below: changing it to an arrow function will make `this` bind to the outer level’s `this` instead, which refers to the `module.exports` object.